Support path exclusion from basic authentication (#106)
prometheus/exporter-toolkit: docs/web-configuration.md

# required. Passwords are hashed with bcrypt.
basic_auth_users:
  [ <string>: <secret> ... ]

> # Exclude URL path from basic authentication. One of the reasonable usecase
> # would be exclude the health check path when basic_auth_users is configured.

Reviewer: Wording suggestion:

```suggestion
# Exclude URL path from basic authentication. For example, health check or
# metrics endpoints.
```

Author: Done. Thank you :)
